### DISCUSSION TITLE
Can same buffer be used in multiple concurrent SEND_ZC ops?

### DISCUSSION THREAD

**Discussion by redbaron** [axboe/liburing#800] [Q&A] [answered]
In a pubsub system it is often happens that same buffer has to be sent to mutliple connections.

Does SEND_ZC support buffer reuse in multiple SQEs concurrently? what about fixed buffers?

**Comment by isilence** [comment, marked as answer]
It's fine as long as nobody modifies the buffer until all requests are completed with their notifications.
